client-course-schedulizer/src/utilities/helpers/caseFunctions.ts

```diff
@@ -127,17 +127,17 @@ export const anticipatedSizeCallback = (value: string, { section }: CaseCallback
 };
 
 export const commentsCallback = (value: string, { section }: CaseCallbackParams) => {
-  section.comments = value;
+  if (value.trim() !== "") {
+    section.comments = value;
+  }
 };
 
 export const yearCallback = (value: string, { section }: CaseCallbackParams) => {
   section.year = yearCase(value);
 };
 
 export const studentHoursCallback = (value: string, { section }: CaseCallbackParams) => {
-  section.studentHours = value.startsWith("$")
-    ? integerDefaultZeroCase(value.substr(1))
-    : integerDefaultZeroCase(value);
+  section.studentHours = value.startsWith("$") ? Number(value.substr(1)) : Number(value);
```

Comment:
why do we check for $? perhaps adding a comment would be good since it isn't immediately apparent.